projects/plugins/jetpack/modules/widgets/migrate-to-core/image-widget.php

```diff
@@ -176,9 +185,11 @@ function jetpack_migrate_image_widget() {
 		}
 
 		foreach ( $sidebars_widgets as $sidebar => $widgets ) {
+			$key = is_array( $widgets ) ? array_search( "image-{$id}", $widgets, true ) : false;
+
 			if (
 				is_array( $widgets )
-				&& false !== ( $key = array_search( "image-{$id}", $widgets, true ) )
+				&& false !== $key
 			) {
```

Comment:
No need to check `is_array()` again now.
```suggestion
		if ( false !== $key ) {
```

Reply:
👍 b7c781e90394caa4c851f6a770e99d7648721e90